.github/workflows/integration-tests.yml

```diff
@@ -5,12 +5,14 @@ on:
     branches: [master]
     paths:
       - ".github/workflows/integration-tests.yml"
+      - "dev/docker-compose.yml"
       - "verification/curator-service/**"
       - "data-serving/data-service/**"
```

Comment:
kinda annoying how I set up this dir structure but we might also want to test on

data-serving/samples

in case someone (me??) updates the sample cases.json and it no longer imports successfully with the schema

Reply:
Good catch, done.